db/seeds.rb

```diff
@@ -1,7 +1,17 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Sponsor.delete_all
+Workshop.delete_all
+Event.delete_all
+
+Workshop.create!([
+  { title: "WS1", description: "D1", image: "image1", link_to_live_demo: "link1", link_to_github: "linkg1", },
+  { title: "WS2", description: "D2", image: "image2", link_to_live_demo: "link2", link_to_github: "linkg2", },
+  { title: "WS3", description: "D3", image: "image3", link_to_live_demo: "link3", link_to_github: "linkg3", },
+])
+
+Sponsor.create!([
+  { company_name: "WS1", logo: "D1", },
+])
+
+event1 = Event.create!(title: "TOC 1", image: "Image 1", location: "Amsterdam", spaces_available: 1, workshop_id: 1, status: true, sponsor_id: 1)
```

Comment:
don't use `workshop_id: 1`

Create workshop, and assign them to a variable:
but use Rails' magic: `workshop: workshop_1`

Or like this: 
`workshop: Workshop.first`